.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Bug report
+about: Report a bug in Quarkus
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+(Describe the problem clearly and concisely.)
+
+**Expected behavior**
+(Describe the expected behavior clearly and concisely.)
+
+**Actual behavior**
+(Describe the actual behavior clearly and concisely.)
+
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. 
+2. 
+3. 
+
+**Configuration**
+```properties
+# Add your application.properties here, if applicable.
+
+```
+
+**Screenshots**
+(If applicable, add screenshots to help explain your problem.)
+
+**Environment (please complete the following information):**
+ - Output of `uname -a` or `ver`: 
```

Comment:
should we qualify which one to be run on each OS ?

Reply:
I thought about it, but I figure it won't take long for people to figure it out.  This is a bug report after all, so it will be filed by a developer who (theoretically at least) knows how to use a computer.

Reply:
I'm not entirely convinced that a Windows user will know to not try "uname" but I guess it's good to have a selection process about who we want to hear issues from :)

Reply:
Does it really matter if they try it? :)